finalProject/ceasarCipher.js

```diff
@@ -0,0 +1,20 @@
+function ceasarCipher(string, string2, num){
```

Comment:
Great job coding this out! What happens if a string with spaces is passed through?  One such as `"Hello World"` or `"Foo Bar Baz"`

Reply:
@mschae16 I hadn't thought about that. It will produce undefined since there is space built into my cipher string to pull from. I would need to rework my code a bit to account for that!